cli/src/main/resources/com/liferay/blade/cli/util/dependencies/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This should all be added to the custom template not in `cli`

Reply:
Agreed :+1: